<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1369</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MS Library Matchmaker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Records</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="numRecords">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="fromTable">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Review</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="numMatches">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="foundLabel">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="nextButton">
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>18</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Next</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="reviewFilterEnable">
                 <property name="text">
                  <string>Filter:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="reviewFilterSelector"/>
               </item>
               <item>
                <widget class="QLineEdit" name="reviewFilterValue"/>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Â±</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="reviewFilterTolerance">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="maximum">
                  <double>999.990000000000009</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTreeWidget" name="repTable">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectItems</enum>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <property name="expandsOnDoubleClick">
                <bool>false</bool>
               </property>
               <attribute name="headerShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string notr="true">1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="matchControlsLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="repThLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Match Threshold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="likelyThreshSB">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>850</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>NIST Th</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="nistThreshSB">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>850</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="use_ri_cb">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Use RI</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>RI Margin</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="ri_marginSB">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>RI Tag</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="RI_tag_cb">
               <item>
                <property name="text">
                 <string>Retention_index</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>n_alkane_RTI</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>d_alkane_RTI</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>UID Prefix</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UID_prefix">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>UID start #</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="UID_start">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="scanButton">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Scan</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="rescanButton">
               <property name="minimumSize">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Re-Scan</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%p%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="testButton">
             <property name="text">
              <string>Test</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Mass Spectrum</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="clearOld_CB">
              <property name="text">
               <string>Clear old matches</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Scan Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="native_RB">
              <property name="text">
               <string>Native</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="nist_RB">
              <property name="text">
               <string>NIST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="both_RB">
              <property name="text">
               <string>Both</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="MatplotlibWidget" name="compoundMS" native="true"/>
            <widget class="QTextEdit" name="compoundTags">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="navBarLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="statusText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="notesText"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1369</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Project_File"/>
    <addaction name="actionSave_Project_File"/>
    <addaction name="actionSave_Project_File_As"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_From_Library"/>
    <addaction name="actionSave_Merged_Library"/>
    <addaction name="actionSave_Library_for_NIST_Search"/>
    <addaction name="actionShow_Record_Counts_3"/>
    <addaction name="actionExport_Matches_2"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionStrike"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_From_Library">
   <property name="text">
    <string>Open Input Libraries...</string>
   </property>
  </action>
  <action name="actionOpen_To_Library">
   <property name="text">
    <string>Open &quot;To&quot; Library...</string>
   </property>
  </action>
  <action name="actionSave_Merged_Library">
   <property name="text">
    <string>Save Merged Library...</string>
   </property>
  </action>
  <action name="actionCreate_Rep_Library">
   <property name="text">
    <string>Create Rep Library...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen_Project_File">
   <property name="text">
    <string>Open Project File...</string>
   </property>
  </action>
  <action name="actionSave_Project_File_As">
   <property name="text">
    <string>Save Project File As...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionStrike">
   <property name="text">
    <string>Strike/Unstrike Record</string>
   </property>
  </action>
  <action name="actionSave_Project_File">
   <property name="text">
    <string>Save Project File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_Matches">
   <property name="text">
    <string>Export Matches...</string>
   </property>
  </action>
  <action name="actionExport_Matches_2">
   <property name="text">
    <string>Export Matches...</string>
   </property>
  </action>
  <action name="actionShow_Record_Counts_3">
   <property name="text">
    <string>Show Record Counts</string>
   </property>
  </action>
  <action name="actionSave_Library_for_NIST_Search">
   <property name="text">
    <string>Save Library for NIST Search...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
